### ISSUE TITLE
Unable to Increase Default Root Device from AWS Launch Template

### ISSUE THREAD

**Issue by asafpelegcodes** [terraform-providers/terraform-provider-aws#4774]
```
Terraform v0.11.7
+ provider.aws v1.22.0
```

I'm testing out creating Launch Templates with aws_launch_template and I'm unable to reproduce a result that I achieved manually through the GUI.  Using the GUI I was able to override the default block device for the AMI listed below (see describe image) and give it a larger volume.  The GUI shows "Default" for this (see image).

![screen shot 2018-06-07 at 10 44 39 am](https://user-images.githubusercontent.com/6590390/41107525-7c95235e-6a40-11e8-917e-14a8b72a18b4.png)

However, when I try & recreate the same thing with terraform it doesn't seem to be possible because even if I omit `delete_on_termination` & `encrypted` terraform provides default values for this.  When I create a spot request I get errors from the console saying `the encrypted flag cannot be specified since device /dev/xvda has a snapshot specified.`

`aws ec2 describe-images --image-ids ami-04351e12` 

Here is the sample terraform config I am working with.
```
resource "aws_launch_template" "ecs-cluster-default" {
...

  block_device_mappings {
    device_name = "/dev/xvda"
    ebs {
      volume_size = 10
      volume_type = "gp2"
      delete_on_termination = "true"
      encrypted = "false"
    }
  }

  image_id = "ami-04351e12"
}
```

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Expected Behavior

I should be able to omit encrypted & delete_on_termination from the block/ebs block and terraform will omit the values to the API and AWS will use Defaults for these blocks.

### Actual Behavior

Terraform forces the value when it's omitted to a default value it has stored internally resulting in being unable to overwrite the root device.

**Comment by cosmopetrich** [comment]
This appears to be a duplicate of #4553.

**Comment by bflad** [comment]
Bug fix pull request submitted: #5632

**Comment by bflad** [comment]
The fix for this has been merged into master and will release with version 1.34.0 of the AWS provider, likely later today.

**Comment by bflad** [comment]
This has been released in [version 1.34.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1340-august-30-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

**Comment by asafpelegcodes** [comment]
@bflad thanks!
